sklearn/utils/_array_api.py

```diff
@@ -735,6 +735,19 @@ def _nanmax(X, axis=None, xp=None):
         return X
 
 
+def _clip(X, S, min_val, max_val, xp):
```

Comment:
Is `X` needed in the parameters to this function?

Reply:
No it's not, thank you for catching it. I will remove this